app/views/admin/tickets/_form.html.haml

```diff
@@ -12,7 +12,10 @@
       = f.input :title, input_html: { autofocus: true }
       = f.input :description, input_html: { rows: 5, data: { provide: 'markdown-editable' } }
       = f.input :price
-      = f.input :price_currency, as: :select, class: 'form-control', collection: ['USD', 'EUR', 'GBP', 'INR', 'CNY', 'CHF'], include_blank: false
+      - if (@conference.tickets.count >= 1 && @ticket.new_record?) || @conference.tickets.count > 1
+        = f.input :price_currency, as: :select, class: 'form-control', collection: [@conference.tickets.first.price_currency], include_blank: false
```

Comment:
this shouldn't be a select with a collection, as you only have one. It should be just shown.

Reply:
I had trouble with form submission without keeping the input. Thinking about it now I guess there's probably an easy way to add a read only input.